pubspec.yaml

```diff
@@ -35,55 +35,55 @@ dependencies:
   # Doesn't support: Linux Mac Windows
   flutter_fimber: ^0.6.5
   # Doesn't support: Linux Windows
-  firebase_core: ^2.9.0
+  firebase_core: ^2.11.0
   # Doesn't support: Linux Mac Windows
-  firebase_dynamic_links: ^5.1.0
+  firebase_dynamic_links: ^5.2.0
   # Doesn't support: Linux Windows
-  firebase_messaging: ^14.4.0
+  firebase_messaging: ^14.5.0
   flutter_bloc: ^8.1.2
   # Doesn't support: Linux Mac Windows
   flutter_fgbg:
     git:
       url: https://github.com/breez/flutter_fgbg.git
       ref: 976b28ff3c299836f215b6deb2003838c49a8001
   flutter_inappwebview: ^6.0.0-beta.22
-  flutter_rust_bridge: 1.75.1
+  flutter_rust_bridge: 1.75.2
```

Comment:
Do we need to change the docs of the flutter_rust_bridge version?

Reply:
Right, I forgot to update the README in breez-sdk.